packages/xo-web/src/xo-app/new/network/index.js

```diff
@@ -99,6 +100,7 @@ const NewNetwork = decorate([
         }
         this.state.networks = networks
       },
+      onChangeNbd: (_, nbd) => ({ nbd: nbd.value }),
```

Comment:
`nbd` can be `null` if we clear the value of the `Select`.
```suggestion
      onChangeNbd: (_, nbd) => ({ nbd: nbd?.value }),
```